app/Http/Controllers/DaylightController.php

```
@@ -55,46 +55,46 @@ public function index(Request $request) {
-            $dayMinutes = 0;
-        } elseif ($sunInfo['sunrise'] === true) {
-            $dayMinutes = 24 * 60;
-        } else {
-            $dayMinutes = ($sunInfo['sunset'] - $sunInfo['sunrise']) / 60;
+            if ($sunInfo['sunrise'] === false) {
+                $dayMinutes = 0;
+            } elseif ($sunInfo['sunrise'] === true) {
+                $dayMinutes = 24 * 60;
+            } else {
+                $dayMinutes = ($sunInfo['sunset'] - $sunInfo['sunrise']) / 60;
+            }
```

PHPMD (2.15.0)

The method index uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)